zerver/lib/alert_words.py

```diff
@@ -36,26 +34,31 @@ def get_alert_word_automaton(realm: Realm) -> ahocorasick.Automaton:
     return alert_word_automaton
 
 def user_alert_words(user_profile: UserProfile) -> List[str]:
-    return ujson.loads(user_profile.alert_words)
+    alert_word_list = []
+    for alert in AlertWord.objects.filter(user_profile=user_profile):
+        alert_word_list.append(alert.word)
+    return alert_word_list
```

Comment:
I think this can be just `return AlertWord.objects.filter(user_profile=user_profile).values_list("word", flat=True)